editor.planx.uk/src/@planx/components/MultipleFileUpload/model.ts

```diff
@@ -1,5 +1,7 @@
+import cloneDeep from "lodash/cloneDeep";
```

Comment:
TIL: The native `structuredClone()` is not yet supported in our version of Jest / JSDOM https://github.com/jsdom/jsdom/issues/3363